python/ray/experimental/state.py

```diff
@@ -332,3 +333,26 @@ def log_files(self):
       ip_filename_file[ip_addr][filename] = file_str
 
     return ip_filename_file
+
+  def task_profiles(self):
+    """Fetch and return a list of task profiles.
+
+    Returns:
+      A list of (task_id, task_profile) pairs and a list of events with no ID.
```

Comment:
This doesn't seem to actually return a list of (task_id, task_profile) pairs. It just returns a list of task_profiles, right?

Shouldn't we make it return a dictionary mapping task IDs to task profiles? Alternatively, we can also leave it in the current form, but we should fix the documentation.